### PR TITLE
Add Chicagoboss

### PR DESCRIPTION
**Pull request by waiteb3**
#1580

### CODE REVIEW

**Comment by k-r-g** [comment]
Awesome! I'm happy to see Chicagoboss added.

File: frameworks/Erlang/chicagoboss/benchmark_config.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+      "plaintext_url": "/plaintext",
+      "json_url": "/json",
+      "port": 8080,
+      "approach": "Realistic",
+      "classification": "Platform",
+      "database": "MySQL",
+      "framework": "chicagoboss",
+      "language": "Erlang",
+      "orm": "Raw",
+      "platform": "cowboy",
```
Comment (msmith-techempower) [inline frameworks/Erlang/chicagoboss/benchmark_config.json]: Cannot help but notice that the classification is `Platform`, but under platform, it says `cowboy`; which is it?

File: frameworks/Erlang/chicagoboss/rebar.config [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{deps, [
```
Comment (msmith-techempower) [inline frameworks/Erlang/chicagoboss/rebar.config]: Adding the comment here, but can you rebase your pull request to NOT have unneeded binary files like `favicon.ico` and `chicago-boss.png`? The project size is already rather large without these sorts of unnecessary things.

**Comment by blee-techempower** [comment]
Local test failed. ( http://sprunge.us/SOdH )
Please resolve this issue.

**Comment by nbrady-techempower** [comment]
metadata fixed, removed unnecessary files and fixed specific ip issue - plaintext and json pass with no warnings - @msmith-techempower

**Comment by msmith-techempower** [comment]
@nbrady-techempower Looks good to me - LGTM!

**Pull request merged**